src/osd/ECBackend.cc

```diff
@@ -2181,7 +2181,13 @@ int ECBackend::objects_read_sync(
   uint32_t op_flags,
   bufferlist *bl)
 {
-  return -EOPNOTSUPP;
+  return store->read(
```

Comment:
see https://github.com/ceph/ceph/pull/37980#discussion_r519450821. i am quoting it here

i don't think we can read a single chunk shard to rebuild the data. might want to refer to `ECBackend::objects_read_async()` for more details on how we do this in the async way.

Reply:
I don't think this will actually work. What we actually need is to add a `read_async` call to CLS that takes a callback.